### DISCUSSION TITLE
Step 2 is not getting updated

### DISCUSSION THREAD

**Discussion by ghost** [skills/.github#261] [Q&A] [answered]
I'm working on the Release based workflow. I've completed the activity mentioned in the step 1, but it's not getting updated to the next step. Someone help me with this.

**Comment by sinsukehlab** [comment, marked as answer]
You have created a pre-release with a **v.09** tag instead of the **v0.9** tag in your fluffy-octo-train repository.

**Reply by ghost** [reply]
Oh, right. Thank you so much for pointing out!

**Reply by sinsukehlab** [reply]
I'm glad I could help out. I would appreciate it if you could mark my reply as the answer.

**Reply by sinsukehlab** [reply]
Thanks for marking my answer.

**Reply by ghost** [reply]
Most Welcome!

**Comment by Harshithanunnaguppala** [comment]
Step 2 is not gettinng updated ! What to do?

**Reply by francisdavey** [reply]
Nor for me. The "Visit" button isn't visible either. Too much magic for me to know what is happening.

**Reply by sinsukehlab** [reply]
Please open a new discussion.
GitHub Actions and GitHub Pages are degraded now.
https://www.githubstatus.com/incidents/ws2l6203cs7g
